godtools/App/Flows/Tract/TractFlow.swift

```diff
@@ -11,12 +11,12 @@ import GodToolsToolParser
 import Combine
 import LocalizationServices
 
-class TractFlow: ToolNavigationFlow, Flow {
+class TractFlow: ToolNavigationFlow, ToolSettingsNavigationFlow {
         
     private let appLanguage: AppLanguageDomainModel
     
-    private var toolSettingsFlow: ToolSettingsFlow?
     private var cancellables: Set<AnyCancellable> = Set()
+    internal var toolSettingsFlow: ToolSettingsFlow?
```

Comment:
Hey @rachaelblue just a side-note, I'm okay with omitting ```internal``` since it's the default access level (https://docs.swift.org/swift-book/documentation/the-swift-programming-language/accesscontrol#Default-Access-Levels).  However, being verbose never hurts either.   

One downside to protocol extension's is when there's a need to mutate at attribute in the extension, end up exposing attributes as in this case.

Reply:
Good point 🤦‍♀️